CLAUDE.md

```
@@ -101,7 +101,18 @@ keyring active is your agent identity. No switch needed for commits.
 5. Post comments on any issues found.
 6. Address each comment via fix commits (commits use git config, no
    gh auth involved — byline stays nathanjohnpayne).
-7. Repeat steps 4–6 until the reviewer identity approves.
+7. Repeat steps 4–6 until the reviewer identity approves. The mechanism
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 4; Actual: 7; Style: 1/2/3

(MD029, ol-prefix)